cms/djangoapps/contentstore/utils.py

```diff
@@ -295,9 +295,10 @@ def get_group_display_name(user_partitions, xblock_display_name):
         group name (String): Group name of the matching group.
     """
     for user_partition in user_partitions:
-        for group in user_partition['groups']:
-            if str(group['id']) in xblock_display_name:
-                return group['name']
+        if user_partition['scheme'] == 'random':
```

Comment:
@cahrens `user_partition` dict has a `scheme` key, so I am using it to only iterate `random` scheme user partitions now. @muzaffaryousaf FYI